### ISSUE TITLE
fair1m Only small part 1 dataset

### ISSUE THREAD

**Issue by robmarkcole** [isaaccorley/torchrs#39]
I noticed the official dataset is split into parts 1 & 2, with the bulk of the images being in part 2

![image](https://user-images.githubusercontent.com/11855322/133425400-d8d30aaa-38c8-4000-a12c-bdd7a4ef2179.png)

The data downloaded using the script in this repo only downloads a subset of 1733 images, which I believe are the part 1 images?

**Comment by isaaccorley** [comment]
That's correct. I haven't updated it in a month or so, so they may have released more imagery.

**Comment by robmarkcole** [comment]
The part2 images are 42.94 GB on disk for 14,757 items. I guess 1 million images will be a gigantic dataset on disk

**Comment by jsxyhelu** [comment]
How do I download the full information at this time?
